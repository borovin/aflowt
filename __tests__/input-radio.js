import test from 'ava'
import Browser from './helpers/browser'
import {prettyPrint} from 'html'

test.beforeEach(t => {
  t.context.browser = new Browser()

  return t.context.browser.goto('/examples/b-input-radio.html')
})

test.afterEach(t => {
  return t.context.browser.close()
})

test('Unchecked radio input', async t => {
  const snapshot = await t.context.browser.snapshot('b-input-radio')

  t.snapshot(prettyPrint(snapshot))
})

test('Checked radio input', async t => {
  const snapshot = await t.context.browser.snapshot('b-input-radio[checked]')

  t.snapshot(prettyPrint(snapshot))
})
