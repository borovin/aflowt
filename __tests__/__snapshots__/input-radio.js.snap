// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checked radio input 1`] = `
"<b-input-radio name=\\"e\\" checked=\\"\\" value=\\"radio 1\\" label=\\"radio button 1\\">
    <label>
        <input value=\\"radio 1\\" name=\\"e\\" type=\\"radio\\" checked=\\"\\">
        <b-input-radio--icon></b-input-radio--icon>
        <b-input-radio--text>radio button 1</b-input-radio--text>
    </label>
</b-input-radio>"
`;

exports[`Unchecked radio input 1`] = `
"<b-input-radio name=\\"e\\" value=\\"radio 2\\" label=\\"radio button 2\\">
    <label>
        <input value=\\"radio 2\\" name=\\"e\\" type=\\"radio\\">
        <b-input-radio--icon></b-input-radio--icon>
        <b-input-radio--text>radio button 2</b-input-radio--text>
    </label>
</b-input-radio>"
`;
